frame/dex-router/src/lib.rs

```diff
@@ -121,10 +138,124 @@ pub mod pallet {
 	}
 
 	#[pallet::call]
-	impl<T: Config> Pallet<T> {}
+	impl<T: Config> Pallet<T> {
+		#[pallet::weight(T::WeightInfo::update_route())]
+		pub fn update_route(
+			origin: OriginFor<T>,
+			asset_pair: CurrencyPair<T::AssetId>,
+			route: Option<BoundedVec<T::PoolId, T::MaxHopsInRoute>>,
+		) -> DispatchResult {
+			let who = ensure_signed(origin)?;
+			let _ = <Self as DexRouter<
+				T::AccountId,
+				T::AssetId,
+				T::PoolId,
+				T::Balance,
+				T::MaxHopsInRoute,
+			>>::update_route(&who, asset_pair, route)?;
+			Ok(())
+		}
+
+		#[pallet::weight(T::WeightInfo::exchange())]
```

Comment:
pallet call docs used to generate scale API and visible in polkadot js. may be add docs in next pr to help qa/integrators